api_bonus.yaml

```
@@ -14,34 +14,8 @@ paths:
       operationId: startBonusEligibilityCheck
       summary: Start bonus eligibility check (ISEE)
       responses:
-        "201":
-          description: Request created.
-          schema:
-            $ref: "#/definitions/InstanceId"
-          headers:
-            Location:
-              type: string
-              description: |-
-                Location (URL) of created request resource.
-                A GET request to this URL returns the request status and details.
-        "202":
-          description: Processing request.
-        "401":
-          description: Bearer token null or expired.
-        "403":
-          description: |
-            Cannot request a new check since there's already
-            an active bonus related to this user
-        "409":
-          description: "Conflict: pending request found."
-          schema:
-            $ref: "#/definitions/ProblemJson"
-        "451":
-          description: Unauthorized for legal reason.
-        "500":
-          description: Service unavailable.
-          schema:
-            $ref: "#/definitions/ProblemJson"
+        "404":
+          description: The feature has been dismissed
```

Maybe we should add the response schema to keep compliant the spec with the real response provided by the API
```suggestion
          description: The feature has been dismissed
          schema:
            $ref: "#/definitions/ProblemJson"
```